browser-interface/packages/shared/dao/types.ts

```diff
@@ -22,6 +22,8 @@ type BaseCandidate = {
   version: { bff: string; comms: string; lambdas: string; content: string }
   elapsed: number
   status: ServerConnectionStatus
+  publicKey: string
+  address: string
```

Comment:
we are not including the address in the about anymore, so maybe we should not include it here?